tests/test_georaster.py

```diff
@@ -409,48 +411,147 @@ def test_masking(self, example: str) -> None:
         with pytest.raises(ValueError, match="mask must be a numpy array"):
             r.set_mask(1)
 
-    @pytest.mark.skip(
-        "Issue: the cropping is off by 30 m for the right bound (where the cropped raster used to be "
-        "larger than the original, so the check was useless)."
-    )  # type: ignore
-    def test_crop(self) -> None:
+    test_data = [[landsat_b4_path, everest_outlines_path], [aster_dem_path, aster_outlines_path]]
+
+    @pytest.mark.parametrize("data", test_data)  # type: ignore
+    def test_crop(self, data: list[str]) -> None:
+
+        raster_path, outlines_path = data
+        r = gr.Raster(raster_path)
+
+        # -- Test with cropGeom being a list/tuple -- ##
+        cropGeom = list(r.bounds)
+
+        # Test with same bounds -> should be the same #
+        cropGeom2 = [cropGeom[0], cropGeom[1], cropGeom[2], cropGeom[3]]
+        r_cropped = r.crop(cropGeom2, inplace=False)
+        assert r_cropped.bounds == r.bounds
+        assert gu.misc.array_equal(r.data, r_cropped.data)
```

Comment:
The `equal_nan` functionality of `np.array_equal` was introduced in NumPy 1.19.0, compatible with Python 3.6.
It was relevant to have this for backward compatibility a year ago when it was relatively new, but now we only use Python >3.6 anyway, and Numpy is at the 1.23 release already.
We should deprecate `gu.misc.array_equal` and switch fully to `np.array_equal` for simplicity. I'm opening an issue.

Reply:
See my comment in #291.